MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -218,6 +219,30 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
     }
 }
 
+// MARK: NavigationMapViewDelegate
+
+extension RouteMapViewController: NavigationMapViewDelegate {
+    
+    func navigationMapView(_ mapView: NavigationMapView, shouldUpdateTo location: CLLocation) -> CLLocation? {
```

Comment:
This is a weird name for a method that returns a CLLocation. The name implies that it returns only a Boolean indicating whether the map view should… update.